webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,999 @@
+            print("error: ", e)
+    return model
+
+
+
+def make_p_values_plot(p_sorted, theme):
+    """Make p-values plot"""
+    p_values = p_sorted.values
+    parameters = p_sorted.index
+    fig = go.Figure()
+    fig.add_trace(
+        {
+            "x": [param.replace("*", "*<br>") for param in parameters],
+            "y": p_values,
+            "type": "bar",
+            "marker":{"color": ["crimson" if val<0.05 else "#606060" for val in p_values]}
+        }
+    )
+    fig["layout"].update(
+        theme_layout(
+            theme,
+            {
+                "barmode": "relative",
+                "height": 500,
+                "title": f"P-values for the parameters. Value lower than 0.05 is statistically significant"
+            }
+        )
+    )
+    fig.add_shape(
+        {
+            "type": "line", 
+            "y0": 0.05, "y1": 0.05, "x0": -0.5, "x1": len(p_values)-0.5, "xref": "x",
+            "line": {"color": "#303030", "width": 1.5}
+        }
+    )
+    fig["layout"]["font"].update({"size": 12})
+    return fig
+
+def make_arrow_plot(coeff_sorted, p_sorted, theme):
+    """Make arrow plot for the coefficients"""
+    params_to_coefs = dict(coeff_sorted)
+    p_values = p_sorted.values
+    parameters = p_sorted.index
+    coeff_vals = list(map(params_to_coefs.get, parameters))
+    sgn = np.sign(coeff_vals)
+    
+    domain = 2
+    steps = domain/(len(parameters)-1) if len(parameters) > 1 else 0
+    num_arrows = len(parameters)
+    x = np.linspace(0, domain, num_arrows) if num_arrows>1 else np.linspace(0, domain, 3)
+    y = np.zeros(len(x))
+
+    fig = px.scatter(x=x, y=y, opacity=0)
+    
+    fig.update_layout(
+        yaxis=dict(range=[-0.15, 0.15], title='', 
+                   showticklabels=False), 
+        xaxis=dict(range=[-0.23, x[-1]+0.23], 
+                   title='', 
+                   ticktext=parameters, 
+                   tickvals=[steps*i for i in range(num_arrows)] if num_arrows>1 else [1]),
+        hoverlabel=dict(
+            bgcolor="white", 
+        )
+    )
+    fig.add_annotation(
+        x=-0.23,
+        y=0,
+        text="Small <br>p-value",
+        showarrow=False
+    )
+    fig.add_annotation(
+        x=x[-1]+0.23,
+        y=0,
+        text="Great <br>p-value",
+        showarrow=False
+    )
+    fig["layout"].update(
+        theme_layout(
+            theme,
+            {
+                "barmode": "relative",
+                "height": 500,
+                "title": "Parameters impact (increase " #Usikker på tittel (særlig det i parentes)
+                         "or decrese) on response and "
+                         "their significance"
+            }
+        )
+    )
+    fig["layout"]["font"].update({"size": 12})
+
+    """Costumizing the hoverer"""
+    fig.update_traces(hovertemplate='%{x}') #x is ticktext
+
+    """Adding arrows to figure"""
+    for i, s in enumerate(sgn):
+        xx = x[i] if num_arrows>1 else x[1]
```

Comment:
i, s, sgn and xx are not very descriptive (or, maybe i is since it is the output of enumerate). Can make some slightly more descriptive variable names.